### PR TITLE
conjure-up-2.2-beta4

### PR DESCRIPTION
**Pull request by adam-stokes**
This brings in improvements utilizing an asyncio library for Juju, better error
handling and logging.

Signed-off-by: Adam Stokes <battlemidget@users.noreply.github.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Additionally, ran:
`brew test conjure-up.rb --devel` (for 2.2-beta4)
`brew test conjure-up.rb` (for 2.1.5)

### CODE REVIEW

**Comment by battlemidget** [comment]
@fxcoudert feel free to throw something at me if I'm still missing steps

**Comment by fxcoudert** [comment]
As you are updating resources, this needs to be a revision for stable.

File: Formula/conjure-up.rb [hunk 1]
```diff
@@ -28,15 +29,30 @@ class ConjureUp < Formula
...
+    url "https://pypi.python.org/packages/ba/dd/bb48f3852a80679a2fd0e7728f763d9b737c068da10b0239bff3a2e07896/ubuntui-0.1.6.tar.gz"
+    sha256 "10310507867122aad5f39f5bd4c478771b07331a3df60f68a9aebb1b7cd22f58"
   end
 
   resource "macumba" do
     url "https://pypi.python.org/packages/76/09/07ac27b7a4bd8511ed3c4b0e16b407ba085323779f917da3be9b0b34e9e7/macumba-0.9.3.tar.gz"
     sha256 "b6b71064f86b6e886b5f23235577b0c1a27df2c3122a65f55a03908cf13a4b06"
   end
 
+  resource "python-dateutil" do
```
Comment (fxcoudert) [inline Formula/conjure-up.rb]: Are these new resources needed? If so they should go in the `devel` block

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by ilovezfs** [comment]
Passing on this for now but feel free to reopen if the requested changes have been made.

**Comment by battlemidget** [comment]
Thanks, we're going to wait for Juju 2.2 GA before doing another PR for conjure-up

**Comment by ilovezfs** [comment]
@battlemidget Sounds good! Thanks :)

**Comment by brettdh** [comment]
@battlemidget any updates on this?

**Comment by battlemidget** [comment]
@brettdh Hey Brett, sorry it is a result of not enough time in the day. I'll try to get to this build today

**Pull request closed without merging**